:construction: WIP: Improve Power panic

3d-gussner opened the pull request:
Add DEBUG_PRINTER_STATE
Fix LCD menus shown or hidden
- Resume for SD and host prints

3d-gussner committed:
Add DEBUG_PRINTER_ACTIVE

gudnimg commented on Firmware/ultralcd.cpp:
```diff
@@ -5731,7 +5731,7 @@ void print_stop(bool interactive, bool unconditional_stop)
     }
 
-    if (card.mounted) {
+    if (card.isFileOpen()) {
```
I agree with this change. Could you create a separate PR for it?

3d-gussner committed:
Fix multiple Z lifts when multiple short power panic
Add DEBUG_UVLO
Fix LCD menus to be hidden while power panic recovery isn't finished

3d-gussner committed:
Use printer_recovering instead eeprom reads

3d-gussner committed:
Fix Resume print for SD and host prints
Add multiple printer states to debug_printer_states
